docs-v2/integrations/productivity.mdx

```diff
@@ -30,7 +30,7 @@ sidebarTitle: Productivity
     <Card title="Monday" href="/integrations/all/monday" color="#68a063" />
     <Card title="Mural" href="/integrations/all/mural" color="#68a063" />
     <Card title="Notion" href="/integrations/all/notion" color="#68a063" />
-    <Card title="Pingboard" href="/integrations/all/pingboard" color="#68a063" />
+    <Card title="Pivotaltracker" href="/integrations/all/pivotaltracker" color="#68a063" />
```

Comment:
you removed pingboard

Reply:
@bodinsamuel just updated my pull request. Thank you 🙏